acceptor/ws_acceptor.go

```diff
@@ -160,6 +170,7 @@ func (w *WSAcceptor) serve(upgrader *websocket.Upgrader) {
 
 // Stop stops the acceptor
 func (w *WSAcceptor) Stop() {
+	w.running = false
```

Comment:
only if the below lines didn't error

Reply:
once you issue the Stop() it should be marked as not running anyway, since there might be an error where it couldn't run close() because it was already closed or other issue, and in that situation, the acceptor would keep getting marked as running, when it really wasn't (if it can't even close itself, it should not have running=true). This matches the existing implementation on the TCPAcceptor